kolibri/utils/main.py

```diff
@@ -176,7 +176,7 @@ def _copy_preseeded_db(db_name, target=None):
                     "home/{}.sqlite3".format(db_name),
                 )
             )
-            shutil.copy(db_path, target)
+            shutil.copyfile(db_path, target)
```

Comment:
If you wanted to try to retain some of the old behavior, you could follow this with a `shutil.copymode` and ignore exceptions. All that `shutil.copy` does is call `shutil.copyfile` then `shutil.copymode`. Or you could use `shutil.copystat` like `shutil.copy2` does.

Reply:
This preseeding was based on the original implementation in the Android installer: https://github.com/learningequality/kolibri-installer-android/blob/master/src/android_service.py#L34 which used `shutil.move` - seemingly that by default uses `shutil.copy` - so if we wanted to be entirely veridical, we could attempt this, but I am not sure that we actually need any of the metadata to be retained. I don't think that the original implementation chose one way or the other for any defined purpose.